app/controllers/application_controller.rb

```diff
@@ -1,2 +1,36 @@
 class ApplicationController < ActionController::API
+
+  before_action :authenticate_user_from_token!
+
+  def authenticate_user_from_token!
```

Comment:
Generally methods that aren't actions on a controller shouldn't be public. While it's unlikely to be accidentally exposed if you have a proper routes file, it's still considered a common convention.